examples/vercel-commerce/e2e/base.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { test, expect } from '@playwright/test';
+import dotenv from 'dotenv';
+import path from 'node:path';
+
+dotenv.config({ path: path.resolve(__dirname, '..', '.dev.vars') });
+
+test('the basic layout of the application works as expected', async ({ page }) => {
+  await page.goto('/');
+
+  await expect(page.getByPlaceholder('Search for products...')).toBeVisible();
+  await expect(page.getByLabel('Open cart')).toBeVisible();
+
+  await expect(page.getByRole('link', { name: 'Created by ▲ Vercel' })).toBeVisible();
+});
+
+test('the actual commerce functionality', async ({ page }) => {
+  test.skip(!process.env.SHOPIFY_STORE_DOMAIN, 'Populate the .dev.vars file to run the test');
+
+  await page.goto('/');
+
+  const hanraShirtLink = page.getByRole('link', { name: 'Hanra Shirt Hanra Shirt $108.' });
```

Comment:
This test is brittle as it depends on the content of the shop.
At least we should test what shop we are serving but even then it makes little sense if we do not control the content of that shop.

Reply:
yes I totally agree that this is quite brittle, I just didn't want to modify the app more than necessary so, with the html in place there I could not find something better to test this functionality

I am happy to remove this or modify the app to be more easily testable, whatever you prefer 👍

Reply:
oh... I just realized, I could very easily mock the network requests.... 🤔 🤷

Reply:
IMO a basic test as the first one is enough for now